GitHub-Test-4/nlp_exercise_1.py

```diff
@@ -0,0 +1,33 @@
+import spacy
+
+nlp = spacy.load('en_core_web_sm')
+
+doc = open('kah-chatgpt-results.txt', 'r', encoding="utf8")
+text = doc.read()
+textStrings = str(text)
+print(textStrings)
+
+words = nlp(textStrings)
+for token in words:
+    print(token.text, token.pos_)
+    # This labels all parts of speech in my entire ChatGPT experiment about Shadow the Hedgehog haikus,
+    # even my requests and suggestions I made as I went.
+
+for ent in words.ents:
+    print(ent.text, ent.start_char, ent.end_char, ent.label_)
```

Comment:
It'd be helpful to comment on what _each_ part of your code does, like you did with the for loop above this.